lsp-bridge.el

```diff
@@ -1123,6 +1134,8 @@ So we build this macro to restore postion after code format."
 (defun lsp-bridge--kill-python-process ()
   "Kill LSP-Bridge background python process."
   (when (lsp-bridge-process-live-p)
+    (when lsp-bridge-remote-enable-kill-process
+      (lsp-bridge-call-sync "close_client"))
```

Comment:
Shoud remove those code if code of close_client move to "cleanup" function.

And don't use lsp-bridge-call-sync, it will block Emacs.